sql/moz-fx-cjms-prod-f3c7/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,68 @@
+WITH aic_flows AS (
+  -- last fxa_uid for each flow_id
+  SELECT
+    flow_id,
+    MIN(flow_started) AS flow_started,
+    ARRAY_AGG(fxa_uid IGNORE NULLS ORDER BY fxa_uid_timestamp DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS fxa_uid,
+  FROM
+    `moz-fx-cjms-prod-f3c7`.cjms_bigquery.flows_v1
+  JOIN
+    EXTERNAL_QUERY("moz-fx-cjms-prod-f3c7.us.cjms-sql", "SELECT flow_id FROM aic")
+  USING
+    (flow_id)
+  WHERE
+    -- only use the last 10 days in stage
+    submission_date >= CURRENT_DATE - 10
```

Comment:
Should this be different for production?

Reply:
yes, updating to 31 days